hicommonwealth/commonwealth
# Group page UI

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/4990

## Description of Changes
- Added community group dashboard UI, accessible from this link `/:scope/members?tab=groups`.
- Added filter behaviors
- Added checks for admin-level features

## "How We Fixed It"
Not a bug

## Test Plan
Since this is a new feature we have to look for any things that seem a bit off

- Visit the `/:scope/members?tab=groups` page for any community, and verify that all elements work correctly
- Visit the same page with admin account and verify you see the create group button (this is only visible to admins)
- Verify the filters group (search filter will filter groups by name and dropdown filter will filter by joined status i.e if you have joined that group or not)

## Deployment Plan
N/A

## Other Considerations
- Didn't connect the API (its still in dev).
